addons/website/views/snippets/s_blockquote.xml

```diff
@@ -7,7 +7,7 @@
         <div class="s_blockquote_content">
             <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
             <footer class="blockquote-footer">
-                <img src="/web/image/website.s_blockquote_default_image" class="s_blockquote_avatar img rounded-circle mr-2" alt=""/>
+                <img src="/web/image/website.s_blockquote_default_image" class="s_blockquote_avatar img rounded-circle mr-2"/>
```

Comment:
if i'm not wrong, during research on alt optimization, we remark, that an alt="" was better than no alt.
Since it means that image is just there to make beautiful and content in the picture is not critical.
In case you remove alt="", it become penalizing, since we don't know if it mean 'no interest' or 'I forgot it'

> Typically, the alt attribute is required; however, there are several cases in which the HTML spec says it is OK to leave the alt attribute's contents empty. The alt attribute is the text equivalent of the image, which means that it should only be empty if the meaning of the image is already encapsulated by the surrounding text.

Reply:
same comment on each other removed alt of course ;) sorry :/